src/commands/test/prepare/calabash.ts

```diff
@@ -56,6 +61,15 @@ export default class PrepareCalabashCommand extends PrepareTestsCommand {
       this.projectDir = this.workspaceDir;
     }
 
+    if (this.config && this.configPath) {
+      throw new Error("Both arguments --config-path and --config (obsolete) were used. Please use only --config-path.")
+    }
+
+    if (this.config && !this.configPath) {
```

Comment:
This could be just `if (this.config) {` now that the above check has been added.